### ISSUE TITLE
How to do a thing

### ISSUE THREAD

**Issue by adriatic** [aurelia-ui-toolkits/assets#2]
![image](https://cloud.githubusercontent.com/assets/2712405/17328926/39c80ba6-588e-11e6-9e7f-b4f1897dee73.png)

> in the code we call the same functions that would be called for lets say the install button in the jspm screen

Of course

---

![image](https://cloud.githubusercontent.com/assets/2712405/17328959/6191dce8-588e-11e6-9639-d4031bd3c55c.png)

> hmm, perhaps we can still use https://github.com/aurelia/cli/blob/master/lib/commands/new/new-application.json, but I'd need to create a proof of concept

We **absolutely** need to use https://github.com/aurelia/cli/blob/master/lib/commands/new/new-application.json file - as otherwise we will not have Rob on our side and will not be able to evolve Monterey as aurelia-cli evolves.

**Comment by adriatic** [comment]
I would still prefer the wizard with the left side bar which allows you to select a project type and then the right pane handling all parameters for a chosen type

![image](https://cloud.githubusercontent.com/assets/2712405/17329157/5d7d10a4-588f-11e6-9401-ffb3b42c70a4.png)

**Comment by adriatic** [comment]
If you can be patient I could create a gist which will demonstrate such wizard (or perhaps better an Aurelia SPA as it would be quicker for me to do it that way).

**Comment by JeroenVinke** [comment]
can you give some examples of project types?

**Comment by adriatic** [comment]
Yes, of course (I should have written "project templates" instead of "types"

![image](https://cloud.githubusercontent.com/assets/2712405/17329464/bfdcfde4-5890-11e6-9e28-f2af2ae64dce.png)

I like to always see which template I selected, while progressing through the more detailed specification that takes place in the "work panel" (RHS)

**Comment by adriatic** [comment]
**THAT** is really the only change I am proposing ... and the new proposed UX can accommodate a huge number of templates - which is what we should be ready to deal with.

Lastly, this wizard design can seamlessly lead to 

![image](https://cloud.githubusercontent.com/assets/2712405/17329642/8aa531e0-5891-11e6-8601-580f81d3b546.png)

**Comment by adriatic** [comment]
Can you do something else and let me create such wizard - knowing that I would be so slow that you might lose your patience?

**Comment by JeroenVinke** [comment]
![image](https://cloud.githubusercontent.com/assets/2189477/17329722/1e83b1b6-58c4-11e6-9b1d-630948f7d0b2.png)

**Comment by JeroenVinke** [comment]
(of course, this should all take place in a popup, not in the entire screen)

That's it, yes (with the full agreement on a pop-up based wizard, yes)

**Comment by JeroenVinke** [comment]
> Can you do something else and let me create such wizard - knowing that I would be so slow that you might lose your patience?

Go ahead. Also, can you look at https://github.com/monterey-framework/monterey/issues/31#issuecomment-236427279? If we can agree on the UI then we can get started

**Comment by adriatic** [comment]
Now, is is clear that I am not going to do this version as it will take me days, while you will have it in hours. So, do me a favor and use Kendo to do this wizard (think about monterey being able to use "electron app with Kendo" template)

**Comment by JeroenVinke** [comment]
@adriatic I'd rather show the screen designs to Rob, and we will know what to improve right away. 

I think we should finish [this screen design](https://cloud.githubusercontent.com/assets/2189477/17329722/1e83b1b6-58c4-11e6-9b1d-630948f7d0b2.png) and improve [this one](https://cloud.githubusercontent.com/assets/2712405/17328926/39c80ba6-588e-11e6-9e7f-b4f1897dee73.png) and add these to the issue we are all subscribed to

**Comment by adriatic** [comment]
> I think we should finish [this screen design](https://cloud.githubusercontent.com/assets/2189477/17329722/1e83b1b6-58c4-11e6-9b1d-630948f7d0b2.png) and improve [this one](https://cloud.githubusercontent.com/assets/2712405/17328926/39c80ba6-588e-11e6-9e7f-b4f1897dee73.png) and add these to the issue we are all subscribed to

I agree - and since you are in that swing already, please continue
